sbi/neural_nets/density_estimators/flow.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from pyknos.nflows import flows
+from torch import Tensor
+
+from sbi.neural_nets.density_estimators.base import DensityEstimator
+from sbi.types import Shape
+
+
+class NFlows(DensityEstimator):
```

Comment:
I am not sure about this class name. How about `NormalizingFlow`? or `NFlowsWrapper`, or `NflowsFlow` :) ?